RELEASE_NOTES

```diff
@@ -4,36 +4,148 @@ Apache IoTDB (incubating) 0.8.0
 
 This is the first official release of Apache IoTDB after joining the Incubator.
 
+
 New Features
 ------------
 
+IOTDB-1     Add Aggregation query
+IOTDB-4     Asynchronously force sync WAL periodically
 IOTDB-5     Support data deletion
-IOTDB-21    Add ChunkGroup offset information in ChunkGroupMetaData
-IOTDB-25    Add some introduction for JMX MBean Monitor in user guide
-IOTDB-29    Multiple Exceptions when reading empty measurements from TsFileSequenceReader
+IOTDB-11    Support start script for jdk 11 on Windows OS
+IOTDB-18    Improve startup script compatible for jdk11
 IOTDB-36    [TsFile] Enable recover data from a incomplete TsFile and continue to write
+IOTDB-37    Add WAL check tool script
+IOTDB-51    Update post-back module to synchronization module
+IOTDB-59    Support GroupBy query
+IOTDB-60    Support Fill function when query
+IOTDB-73    Add REGULAR encoding method for data with fixed frequency
+IOTDB-80    Support custom export file name
+IOTDB-81    Update travis for supporting JDK11 on Windows
+IOTDB-83    Add process bar for import script and show how many rows have been exported
+IOTDB-91    Improve tsfile-spark-connector to support spark-2.4.3
+IOTDB-93    IoTDB Calcite integration
+IOTDB-109   Support appending data at the end of a completed TsFile
+IOTDB-122   Add prepared statement in JDBC
+IOTDB-123   Add documents in Chinese
+IOTDB-130   Dynamic parameters adapter
+IOTDB-134   Add default parameter for client starting script
+Add read-only mode of IoTDB
+New storage engine with asynchronously flush and close data file
+Adding english documents
+Supporting travis + window + jdk8
+Add skipping all UTs: maven integration-test -DskipUTS=true
+Enable users define the location of their thrift compiler
+Add example module
+Add a log appender: put info, warn, error log into one file and disable log_info by default
+Recover when resource file does not exist while tsfile is complete
+
 
 Incompatible changes
 --------------------
 
+Data folder structures
```

Comment:
You'd better to claim clearly be incompatible with WHO?

Reply:
Wat is WHO?

Reply:
He means with which version... I have updated this description

Reply:
+1.